meetings/2018-11-24.md

```diff
@@ -0,0 +1,61 @@
+# Node.js Foundation Technical Steering Committee (TSC) Meeting 2018-10-24
+
+## Links
+
+* **Recording**:  https://www.youtube.com/watch?v=ZeVKDRQfSyU
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/615
+
+## Present
+
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Daniel Bevenius @danbev (TSC)
+* Matteo Collina @mcollina (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Michaël Zasso @targos (TSC)
+* Sakthipriyan Vairamani @thefourtheye (TSC)
+* Rich Trott @Trott (TSC)
+
+## Agenda
+
+### Announcements
+ 
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* buffer: fix crash for invalid index types [#23795](https://github.com/nodejs/node/pull/23795)
+  * removed from the agenda before the meeting
+  
+* fs: behaviour of readFile and writeFile with file descriptors [23433]( https://github.com/nodejs/node/issues/23433)
+  * Behaviour is inconsistent with the documentation.
+  * Read and Write behave differently
+  * PR exists to update documentation
+  * Call for people to get involved in the issue.
+
+### nodejs/TSC
+
+* Tracking issue for updating TSC on Board Meetings [#476](https://github.com/nodejs/TSC/issues/476)
+  * No Myles, skipped for this week
+
+
```

Comment:
Nit: We don't need these many empty lines, right?

Reply:
Looked in the file, not sure while more spaces are shown as there was only one.  Tried deleting/re-adding a single blank line.

Reply:
Ah was looking at version without one of the commits.